themes/manual/pagenav.html

```diff
@@ -5,7 +5,7 @@ <h3>
   <a class="index-link" href="{{ pathto('index') }}">{{ shorttitle }}</a>
 </h3>
 
-{{ version_selector() }}
+{% if project != 'mongodb-meta' %}{{ version_selector() }}{% endif %}
```

Comment:
There are "better" ways of doing this, where "better" means "more complicated and indirect".

This is fine.